src/docs/forms/components/RadioGroup/examples/03-DisabledProperty.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Form, {
+     RadioGroup
+} from 'doj-react-adminlte/Form';
+
+
+export default class DisabledProperty extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            radioGroupDisabled: false
+        };
+    }
+
+    handleChange = (field, value) => {
+        this.setState({
+            [field]: value
+        });
+    };
+
+    handleOnClick = () => {
+        this.setState({
+            radioGroupDisabled: !this.state.radioGroupDisabled
+        });
+    };
+
+    render() {
+        const {radioGroupDisabled} = this.state;
+        const switchOptions = [
+            {label: "On", value: false},
+            {label: "Off", value: true}
+        ];
+        const radiogrouplbl = this.state.radioGroupDisabled ? "Enable" : "Disable" ;
+
+        return (
+            <Form onChange={this.handleChange}>
+                <div className="row">
+                    <div className="col-xs-3">
+                        <RadioGroup label="Switch"
+                                    name="radioGroupDisabled"
+                                    value={radioGroupDisabled}
```

Comment:
radio group value should have its own state. (i.e., it should not disable the component when "Off" option is clicked. Component will only be disabled through toggling the disable/enable button"